R/LSgameofchance.R

```diff
@@ -76,8 +76,8 @@ LSgameofchance   <- function(jaspResults, dataset, options, state = NULL){
   ## fill in the table and the plot
   if (nPlayers == 2 & max(xPoints) < winPoints){
```

Comment:
Sure, I have added the unit test and changed the symbols.

Reply:
1. I would just leave `p()` there for all and they are easy to change if necessary.
2. I add an `nSims` check in the analysis instead of qml, which has more informative content via the error message.
3. I add another check to make sure `winPoints>=1` and fix the boundary issue when `winPoints=1`.

Reply:
1 & 3 is fine

I think that it is better to restrict the nSims in qml - for consistency as it's done with bootstrapping in other analyses + the user doesn't need to see an error message if it's impossible to set it to bad values